Bottom: bc7fd35354fdf26e9a975628e1caadda6f73ba44
Top:    9882b2f7d3e6a449deacac9e6689fdd9ed076262
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-07 09:28:04 +0200

Refresh of add-a-calculator-value-object

---

diff --git a/src/modules/calculator/value.cpp b/src/modules/calculator/value.cpp
index 3f98edb..0411824 100644
--- a/src/modules/calculator/value.cpp
+++ b/src/modules/calculator/value.cpp
@@ -38,10 +38,10 @@ public:
   constexpr tvalue(int64_t value) noexcept : value_(value) {}
 
   constexpr void set(int64_t value) noexcept { value_ = value; }
-  constexpr int64_t get() const noexcept { return value_; }
+  [[nodiscard]] constexpr int64_t get() const noexcept { return value_; }
 
   /** @returns The formatted output of the value. */
-  std::string format() const { return std::to_string(value_); };
+  [[nodiscard]] std::string format() const { return std::to_string(value_); };
 
 private:
   int64_t value_{0};
